C/Sorting/Shell_sort.c

```diff
@@ -0,0 +1,37 @@
+#include<stdio.h>
+#include<stdlib.h>
+
```

Comment:
explain approach here

Reply:
@zugzwang03 The code you mentioned is of shell sort for which I have created separate PR and have done the modifications there but that PR is not merged till now. This issue is of Circular Linked list for which I have added the code.

Reply:
creating branches would have been better though